pageserver/src/walredo.rs

```diff
@@ -867,10 +865,7 @@ impl PostgresRedoManager {
                 nwrite += proc.stdin.write(&writebuf[nwrite..])?;
```

Comment:
I wonder if the next step will be to add `nwrite` as context to this?

```suggestion
                nwrite += proc.stdin.write(&writebuf[nwrite..])
                    .with_context(|| format!("write {nwrite}..{}", writebuf.len()))?;
```

Reply:
Well, perhaps not, feel free not to include. I misunderstood pipe(7).

Reply:
One step at a time :)